src/porepy/numerics/fv/tpfa.py

```diff
@@ -266,3 +270,462 @@ def discretize(self, sd: pp.Grid, data: dict) -> None:
         matrix_dictionary[
             self.bound_pressure_vector_source_matrix_key
         ] = bound_pressure_vector_source
+
+
+class DifferentiableTpfa:
+    """Helper class for constructing differentiable operators for the TPFA scheme.
+
+    This is really a container class for methods that are needed to construct the
+    discretization. The actual discretization is considered a constitutive law, and can
+    be found in that part of the code.
+
+    """
+
+    def boundary_filters(
```

Comment:
Should we rename to external_boundary_filters?

Reply:
Yes!